ImageKitSample/Program.cs

```diff
@@ -0,0 +1,385 @@
+﻿using Imagekit.Models;
+using Imagekit.Sdk;
+using System;
+using System.Collections.Generic;
+using Imagekit;
+using static Imagekit.Models.CustomMetaDataFieldSchemaObject;
+using Newtonsoft.Json.Linq;
+
+namespace ImagekitSample
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            //Create Instance of ImageKit
+            ImagekitClient imagekit = new ImagekitClient("TestPublicKey", "TestPrivateKey", "https://api.imagekit.io/");
+
+
+            #region URL Generation
+
+            /// Generating URLs
+            string imageURL = imagekit.Url(new Transformation().Width(400).Height(300))
+                .Path("/default-image.jpg")
+                .UrlEndpoint("https://ik.imagekit.io/your_imagekit_id/endpoint")
+                .TransformationPosition("query")
+                .Generate();
+            Console.WriteLine("Url for first image transformed with height: 300, width: 400 - {0}", imageURL);
+
+
+            ///// Generating Signed URL
+            //var imgURL1 = "https://ik.imagekit.io/demo/default-image.jpg";
+            //string[] queryParams = { "b=123", "a=test" };
+            //try
+            //{
+            //    var signedUrl = imagekit.Url(new Transformation().Width(400).Height(300))
```

Comment:
As discussed, please ensure that the order of operations in the sample project is the same as that of the readme.md file.